package.json

```
@@ -20,7 +20,7 @@
   "scripts": {
     "ci-checks": "eslint . && prettier --check . && npm run type-check && npm run deps-check",
     "deps-check": "knip",
-    "prepare": "husky install",
+    "prepare": "husky",
```

Should we change it to?

```suggestion
    "prepare": "husky || true",
```

Ref: https://typicode.github.io/husky/how-to.html#ci-server-and-docker

---

after discussing it, we're keeping the original prepare script